src/ui/js/Overview.js

```diff
@@ -340,6 +347,27 @@ Overview.pageAddGameTable = function(
     i += 1;
     tableBody.append(gameRow);
   }
+
+  if (staleGamesExist) {
+    var tableFoot = $('<tfoot>');
+    var footRow = $('<tr>');
+    var footCol = $('<td>');
+    // james: Do not use colspan="0" because this is incompatible with older
+    //        browsers. Instead, use overkill and specify 100 columns.
+    footCol.attr('colspan', 100);
```

Comment:
I'm suspicious of this --- every other use of colspan in the JS is a number under 10, usually 2, 4, or 6, and is specified as a string (that last one is a nit).  Why is this one 100?

What is the visual effect you are trying to accomplish by setting colspan?  The comment doesn't say at all, and it's not obvious to me.